source/watchdog.py

```diff
@@ -54,6 +54,10 @@ def __getattr__(attrName: str) -> Any:
 """ Seconds before the core should be considered severely frozen and a warning logged.
 """
 
+_CRASH_STATS_FILENAME = "nvda_crash_stats.txt"
+_CRASH_STATS_WINDOW_SEC = 60
+_CRASH_STATS_MAX_COUNT = 3
```

Comment:
I think it's possible NVDA may take longer than 20s to start up on old machines. That means this count may never be reached and the cycle never broken.

```suggestion
_CRASH_STATS_WINDOW_SEC = 120
_CRASH_STATS_MAX_COUNT = 3
```

Reply:
Rolling this into the dataclass, rejecting suggestion here but integrating downstream.